swift/vapor-framework/Package.resolved

```diff
@@ -0,0 +1,133 @@
+{
+  "object": {
```

Comment:
I agree we should remove the resolved file so that as performance fixes go out this benchmark will get them automatically.